org.codeblocks.codeblocks.json

```diff
@@ -0,0 +1,106 @@
+        "--socket=x11",
+        "--socket=wayland",
+        "--share=network",
+        "--filesystem=home"
+    ],
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/share/aclocal",
+        "/share/doc",
+        "/share/man",
+        "*.a"
+    ],
+    "modules": [
+        {
+            "name": "libXmu",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.x.org/releases/individual/lib/libXmu-1.1.2.tar.gz",
+                    "sha256": "e5fd4bacef068f9509b8226017205040e38d3fba8d2de55037200e7176c13dba"
+                }
+            ]
+        },
+        {
+            "name": "libXaw",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.x.org/releases/individual/lib/libXaw-1.0.13.tar.gz",
+                    "sha256": "7e74ac3e5f67def549722ff0333d6e6276b8becd9d89615cda011e71238ab694"
+                }
+            ]
+        },
+        {
+            "name": "xterm",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://invisible-mirror.net/archives/xterm/xterm-331.tgz",
+                    "sha256": "9ae856a30fd93046be93952a6898ba47f6f88ad6a988a7c949c4c80d5199ef10"
+                }
+            ]
+        },
+	{
+	    "name": "wxWidgets",
+	    "rm-configure": true,
+            "cleanup": [
+                "/bin",
+                "/lib/wx",
+                "/share/bakefile"
+            ],
+	    "sources": [
+		{
+		    "type": "archive",
+		    "url": "https://github.com/wxWidgets/wxWidgets/releases/download/v3.0.4/wxWidgets-3.0.4.tar.bz2",
+		    "sha256": "96157f988d261b7368e5340afa1a0cad943768f35929c22841f62c25b17bf7f0"
+		},
+		{
+		    "type": "script",
+		    "dest-filename": "autogen.sh",
+		    "commands": [
+			"cp -p /usr/share/automake-*/config.{sub,guess} .",
+			"autoconf -f -B build/autoconf_prepend-include"
+		    ]
+		}
+	    ]
+	},
+        {
+            "name": "codeblocks",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://netcologne.dl.sourceforge.net/project/codeblocks/Sources/17.12/codeblocks_17.12.tar.xz",
+                    "sha256": "13881a0a72769694e82e531b8e7814d51fbf1fa122c73c5004e186560fbc57e0"
+                },
+                {
+                    "type": "patch",
+                    "path": "codeblocks-icon-name-for-mimetype.patch"
+                },
+                {
+                    "type": "patch",
+                    "path": "codeblocks-appdata-fields.patch"
+                }
+            ],
+            "post-install": [
+                "mv /app/share/mime/packages/codeblocks.xml /app/share/mime/packages/org.codeblocks.codeblocks.xml",
+                "install -D /app/share/pixmaps/codeblocks.png /app/share/icons/hicolor/64x64/apps/codeblocks.png",
```

Comment:
Can remove the left over icon in `/app/share/pixmaps`. Also wrong permissions.

Reply:
Should be fixed now